src/Unicode-TOU.xml

```diff
@@ -0,0 +1,174 @@
+<SPDX name="Unicode Terms of Use" identifier="Unicode-TOU" osi-approved="false">
+  <urls>
+    <url>http://www.unicode.org/copyright.html</url>
+  </urls>
+  <license>
+    <title>
+      <p>Unicode Terms of Use</p>
+    </title>
+    <optional>
```

Comment:
agree this should not be optional. seems intended to have the legal effect of incorporating the privacy policy and tm policy. Whether it's successful, I'm not sure, but I would say don't mark as optional unless we empirically find we miss matches for licenses in the wild based on this.